test/functional/tools/explicit_conversion.xml

```diff
@@ -0,0 +1,23 @@
+<tool id="explicit_conversion" name="explicit_conversion">
+  <command>
+    cut -f 1 '${input1_table}' > 'col_output'
```

Comment:
I made the change but I'm not sure I like it. I get the extra copy is sort of the "default" behavior. But I'm not sure it should be thought of that way - when using various object stores or Pulsar or Docker or "run as real user" or that workflow language you hate the way outputs are copied around are subtlety different. I'd imagine one day all outputs are copied the way ``from_work_dir`` is - it is just too insecure and error prone to have every worker node writing the global object store. I feel like conceptual simplicity should be deciding factor - and there I think that ``from_work_dir`` might be easier to think or more pure since it involves less templating. This isn't clear cut though.